.gitignore

```diff
@@ -9,3 +9,4 @@
 node_modules/
 .DS_STORE
 yarn-error.log
+.yarnrc
```

Comment:
This shouldn't be .gitignored or deleted; it's intentional that it's checked in. See the comment in .yarnrc for the reasoning.

Reply:
I missed that. Updated to restore it back.